### ISSUE TITLE
training chinese with char or phone 

### ISSUE THREAD

**Issue by Seventhen** [guoying1030/tlspeech#1]
hi, guoying
I'd like to  use this project to training a small dataset like ths30. I have just readed your code. I have a question. 
How can I use chinese char to train, I saw your softmax output was alphabet "ABC...". How can I acheive it to training chinese? As I know, aeon.dataloader dos't support char.So I can't to char as alphabet, may be phone? Can you offer me some advice?
thx
